brainstormforce/ultimate-addons-for-gutenberg
# Fix: Table of content - Conflicting with astra mega menu

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/77c500b4-04c6-4def-8e64-457f96e6dd83?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots

- Testing Evidance - https://share.getcloudapp.com/RBuyGNyR

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Step to reproduce the issue -
- Create a mega menu using Astra and Astra Addon. Refer to this [doc ](https://wpastra.com/docs/nav-menu-addon/).
- Create a page with TOC. Publish the page and hover on the mega menu. You will see the mega menu is not opening on hover.
- Switch to the above fix branch, refresh the page, and hover on the mega menu to resolve the issue.

### Checklist:
